sdk_contrib/async_processor/src/main/java/io/opentelemetry/sdk/contrib/trace/export/DisruptorEventQueue.java

```diff
@@ -190,11 +247,13 @@ public void onEvent(DisruptorEvent event, long sequence, boolean endOfBatch) {
             spanProcessor.onEnd(readableSpan);
             break;
           case ON_SHUTDOWN:
+            spanProcessor.forceFlush();
```

Comment:
@bogdandrutu is this correct?

Reply:
It depends where do you want to have the shutdown logic implemented. If we just enqueue a shutdown type event then this is correct, if we do call explicitly shutdown from the thread that does queuing then this is not needed.